app/components/sessions/forms/edit-session.hbs

```diff
@@ -15,11 +15,13 @@
   <div class="auk-u-mb-2">
     <div class="auk-form-group">
       <Auk::Label>{{t "meeting-number"}}</Auk::Label>
-      <WebComponents::VlFormInput
-        data-test-edit-session-meeting-number={{true}}
+      <Auk::Input
         @type="number"
+        @block={{true}}
         @value={{this.meetingNumber}}
-        {{on "input" (action "meetingNumberChangedAction" this.meeting)}}
```

Comment:
Wat is de exacte reden voor het vervangen van dit event? Vermoedelijk wil je het `input`-event behouden, maar de one-way binding `value` (ipv `@value`) gebruiken. Zie ook het 2e punt in https://api.emberjs.com/ember/release/classes/Ember.Templates.components/methods/input#input-html-attributes-to-avoid

Reply:
Ik heb enkele kleine aanpassingen binnen het event gedaan. Is dit wat meer in de trend van wat het zou moeten zijn?

Reply:
En daarmee is ook een issue op dev opgelost waar bij het invoeren van een nieuw meeting nummer de aanpassing niet correct door kwam in het veld daar onder:

<img width="882" alt="CleanShot 2021-08-25 at 11 57 58@2x" src="https://user-images.githubusercontent.com/18174827/130770565-f3e441b8-b01b-4631-923f-086490efc984.png">

Reply:
Zal dit eens nakijken. Wat er vroeger stond is dat we het **model** `meeting` meegaven, maar dat is niet wat we verwachtten in de action, we willen daar een nummer hebben.

Reply:
voor context: @brenner-company heeft ook https://kanselarij.atlassian.net/browse/KAS-2857 aangemaakt

Reply:
Ik weet niet goed wat die `this.meeting` daar had te zoekenen of het ervoor dan correct werkte.
Maar zoalshet nu staat is het goed.

FYI, die bug is opgelost door lijn 1 code te verwijderen.